python/cudf/cudf/core/column/column.py

```diff
@@ -877,20 +886,12 @@ def all(self, skipna: bool = True) -> bool:
         # The skipna argument is only used for numerical columns.
         # If all entries are null the result is True, including when the column
         # is empty.
-        if not isinstance(skipna, bool):
```

Comment:
This (and for `any`) were added to pass some pandas unit tests https://github.com/rapidsai/cudf/pull/20436